jest/jest.d.ts

```diff
@@ -1,126 +1,88 @@
-
-declare function xdescribe(name: string, fn: jest.EmptyFunction): void;
-declare function xit(name: string, fn: jest.EmptyFunction): void;
-
+declare function afterEach(fn: () => any): void;
+declare function beforeEach(fn: () => any): void;
+declare function describe(name: string, fn: () => any): void;
 declare function expect(actual: any): jest.Matchers;
+declare function it(name: string, fn: () => any): void;
+declare function fit(name: string, fn: () => any): void;
 
-interface NodeRequire {
-    requireActual(moduleName: string): any;
-}
+declare function test(name: string, fn: () => any): void;
+declare function xdescribe(name: string, fn: () => any): void;
+declare function xit(name: string, fn: () => any): void;
 
 declare namespace jest {
-    function addMatchers(matchers: CustomMatcherFactories): void;
-    function autoMockOff(): void;
-    function autoMockOn(): void;
-    function clearAllTimers(): void;
-    function currentTestPath(): string;
-    function disableAutomock(): void;
-    function fn<T>(implementation?: Function): Mock<T>;
-    function dontMock(moduleName: string): void;
-    function genMockFromModule<T>(moduleName: string): Mock<T>;
-    function mock(moduleName: string, factory?: Function): void;
-    function runAllTicks(): void;
-    function runAllTimers(): void;
-    function runOnlyPendingTimers(): void;
-    function setMock<T>(moduleName: string, moduleExports: T): void;
-    function unmock(moduleName: string): void;
-
-    interface EmptyFunction {
-        (): void;
-    }
-
     interface Matchers {
+        lastCalledWith(...args: any[]): boolean;
         not: Matchers;
-        toThrow(expected?: any): boolean;
-        toThrowError(expected?: any): boolean;
         toBe(expected: any): boolean;
-        toEqual(expected: any): boolean;
+        toBeCalled(): boolean;
+        toBeCalledWith(...args: any[]): boolean;
+        toBeCloseTo(expected: number, delta: number): boolean;
+        toBeDefined(): boolean;
         toBeFalsy(): boolean;
-        toBeTruthy(): boolean;
+        toBeGreaterThan(expected: number): boolean;
+        toBeGreaterThanOrEqual(expected: number): boolean;
+        toBeLessThan(expected: number): boolean;
+        toBeLessThanOrEqual(expected: number): boolean;
         toBeNull(): boolean;
-        toBeDefined(): boolean;
+        toBeTruthy(): boolean;
         toBeUndefined(): boolean;
-        toMatch(expected: RegExp): boolean;
         toContain(expected: string): boolean;
-        toBeCloseTo(expected: number, delta: number): boolean;
-        toBeGreaterThan(expected: number): boolean;
-        toBeLessThan(expected: number): boolean;
-        toBeCalled(): boolean;
-        toBeCalledWith(...args: any[]): boolean;
-        lastCalledWith(...args: any[]): boolean;
+        toEqual(expected: any): boolean;
+        toMatch(expected: RegExp): boolean;
+        toMatchSnapshot(): boolean;
+        toThrow(): boolean;
```

Comment:
As far as I understood the documentation `toThrow` just checks if function invocation threw error. And `toThrowError` could be used to test Error details, like constructor and Error message. Correct me If I'm wrong here. After all  `toThrow` is still in the API Reference: https://facebook.github.io/jest/docs/api.html#tothrow despite the PR you reference.